espanol/thrones/daynerys.md

```diff
@@ -0,0 +1,3 @@
+La reina hablo, dio un discurso como nunca antes se habia oido, todos los hombres y mujeres presentes estuvieron completamente deacuerdo con ella, y asi la
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.